dom0/sd-log-reboot.sls

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there a reason you've put this in a separate file? Because there's to `requires` here, this means that `sd-log` will be rebooted at a time that is likely not the time you are expecting here

Reply:
Yes, this is where I need help, how to make sure that we reboot `sd-log` in proper time, may be via an instruction in documentation and remove this totally.